packages/salesforcedx-vscode-core/src/notifications/progressNotification.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as vscode from 'vscode';
+import { nls } from '../../src/messages';
+
+import { CommandExecution } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+
+export class ProgressNotification {
+  public static async show(
+    execution: CommandExecution,
+    token: vscode.CancellationTokenSource
+  ) {
+    await vscode.window.withProgress(
```

Comment:
vscode.window.withProgress() is async, so I had to update the `execute()` and `attachExecution()` methods to also be async.